src/vmm/src/config.rs

```diff
@@ -104,11 +108,8 @@ impl TryFrom<String> for KernelConfig {
     fn try_from(kernel_cfg_str: String) -> result::Result<Self, Self::Error> {
         // Supported options:
         // `cmdline=<"string">,path=/path/to/kernel,himem_start=<u64>`
-        // Required: all
+        // Required: path
         let options: Vec<&str> = kernel_cfg_str.split(',').collect();
```

Comment:
```suggestion
        let options: Vec<&str> = kernel_cfg_str.split(',').filter(|tok| !tok.is_empty()).collect();
```

Otherwise, we panic if some option is an empty string, for example one comma at the end:

```bash
cargo run --                                                               
              --memory size_mib=1024 
              --vcpu num=1                     
              --kernel path=vmlinux-hello-busybox,himem_start=1024,cmdline="console=ttyS0 i8042.nokbd reboot=k panic=1 pci=off",
```